package.json

```diff
@@ -3,45 +3,23 @@
   "version": "2.0.1",
   "description": "Grafana data source for Azure Data Explorer",
   "scripts": {
-    "build": "node --max-old-space-size=8192 ./node_modules/.bin/webpack --config webpack.config.prod.js",
-    "dev": "webpack --mode development",
-    "watch": "webpack --mode development --watch",
-    "test": "jest --config jest.config.js",
-    "test-watch": "jest --watch --config jest.config.js"
+    "build": "grafana-toolkit plugin:build && find src | grep '\\.d\\.ts$' | xargs rm -fv",
```

Comment:
Why the find/grep/xargs, and what do they do?

Reply:
Yeah, about that... The toolkit leaves a bunch of .d.ts files in the incoming folder... on the canary version anyway. ???

Reply:
Hmm, weird, that didn't happen in the perspective-panel repo, and it's tsconfig is mostly the same as yours.